blake256_test.go

```diff
@@ -5,13 +5,15 @@
 // worldwide. This software is distributed without any warranty.
 // http://creativecommons.org/publicdomain/zero/1.0/
 
-package blake256
+package blake256_test
```

Comment:
Don't rename please.

Reply:
Will revert if you really want me to. Care to mention why, though?

Reply:
I just like the simplicity of it, I guess 🤷‍♂

Reply:
Well, this way tests cannot change code internals, which makes them more effective (blackbox vs. whitebox testing, basically). But again, I'm happy to revert if you really don't like it.

Reply:
Sure, but it's a few letters more to type for a theoretical benefit! Reading the code is the best guarantee that it can't change internals :) Whatever, let's do a separate package test if you want like.

Reply:
Actually, then it would have hardcoded package path, so let's not do it.

Reply:
Interestingly, the mistake with import in PR *was* caused by a having a separate test package, so I guess my kind of testing wins :)

Reply:
As you wish.